crates/router/src/core/payments/flows/authorize_flow.rs

```diff
@@ -61,12 +61,12 @@ impl Feature<api::Authorize, types::PaymentsAuthorizeData> for types::PaymentsAu
         mut self,
         state: &AppState,
         connector: &api::ConnectorData,
-        maybe_customer: &Option<domain::Customer>,
+        _maybe_customer: &Option<domain::Customer>,
         call_connector_action: payments::CallConnectorAction,
-        merchant_account: &domain::MerchantAccount,
```

Comment:
the merchant account is also not needed, we can remove these from the function params